server/api/person/person.validation.js

```diff
@@ -1,15 +1,24 @@
 const websiteRegex = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w.-]+)+[\w\-._~:/?#[\]@!$&'()*+,;=.]+$/
+const domainRegex = /(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]/
```

Comment:
person.validation and organisation.validation appear to be the same file.  Can we have just one called fieldValidation.js?

Reply:
@avowkind oh where would "fieldValidation.js" be stored. 
I forgot to remove the domain regex on person.validation.js
But would it be better to keep the person.validation and organsation.validation separate ????

Reply:
Let me know what you think  @avowkind

Reply:
the rules should be the same for every form that uses them so put them all together in one validation file.

Reply:
> @avowkind oh where would "fieldValidation.js" be stored.
/lib is for stuff everyone uses.